src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1356,15 +1356,15 @@ def report_timeout(self):
             # dump engines ULT stacks upon test timeout
             self.dump_engines_stacks("Test has timed-out")
 
-    def fail(self, message=None):
+    def fail(self, msg=None):
```

Comment:
(pylint-arguments-renamed) Parameter 'msg' has been renamed to 'self' in overridden 'TestWithServers.fail' method